openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/EntityDetails.spec.js

```diff
@@ -161,6 +163,31 @@ describe('Entity Details Page', () => {
 
     cy.get('[data-testid="entity-tags"]').should('contain', 'Tier1');
 
+    // add tag to the entity
+    interceptURL('GET', '/api/v1/tags?limit=1000', 'tagsRequest');
+    interceptURL(
+      'GET',
+      '/api/v1/search/query?q=*&from=0&size=1000&index=glossary_search_index',
+      'glossaryRequest'
+    );
+
+    cy.get('[data-testid="edit-button"]').should('be.visible').click();
+
+    cy.get('[data-testid="tag-selector"]')
+      .scrollIntoView()
+      .should('be.visible')
+      .type(entityTag);
+
+    verifyResponseStatusCode('@tagsRequest', 200);
+    verifyResponseStatusCode('@glossaryRequest', 200);
+
+    cy.get('.ant-select-item-option-content')
+      .first()
+      .should('be.visible')
```

Comment:
I don't think we are verifying case we have fixed in this PR?
Add a Tire
Add some tags
Remove Tire should not remove tags right?

Are we testing same here?

Reply:
@chirag-madlani , We are testing the same behavior.

- Add the tier, tag
- Remove the tie
- Check for entity tags and they should be present